spec/controllers/applications_controller_spec.rb

```diff
@@ -126,11 +126,13 @@
           id: application.id,
           user:
           {
+            email_for_google: :email_for_google,
             application_attributes: application_params,
             profile_attributes: profile_params
           }
         }
       }
+      let(:email_for_google) { "lemurs@gmail.com" }
```

Comment:
I would add a new example to verify that the email does indeed get saved as you intend, (or add an additional assertion in the example that starts on line 145), something like:
```ruby
it "saves the email for google" do
  expect { subject }.to change { application.profile.email_for_google }.to(email_for_google)
end
```
(code written after 7 PM off the top of my head, possibly not even syntactically correct, but hopefully you get the gist of what I'm thinking 😅)

Reply:
Good plan -- done.

The updated test catches the use of an `:email_for_google` symbol instead of the `email_for_google` variable that you mentioned above.